core/storage/story/gae_models.py

```diff
@@ -58,6 +58,8 @@ class StoryModel(base_models.VersionedModel):
     # The schema version for the story_contents.
     story_contents_schema_version = (
         ndb.IntegerProperty(required=True, indexed=True))
+    # The topic id to which the story belongs.
+    belongs_to_topic = ndb.StringProperty(indexed=False, required=False)
```

Comment:
@seanlip. Will this cause issues with existing stories in the datastore?

Reply:
No, I think it's fine. They'll just be set to None.

But I think indexed should be True. And once we've populated the story models on the test server correctly, we should make this required=True (is that right, @DubeySandeep? If so, we should add a TODO to do this.)

Reply:
For now, I've changed both of the value to True once we will deploy the new branch to testserver we can add topic_id manually before start testing. I'll add an email (with the story id and corresponding topic id) to Sean for the same (for tracking).